src/Nethermind/Nethermind.AuRa.Test/Reward/AuRaRewardCalculatorTests.cs

```diff
@@ -83,6 +90,18 @@ public void constructor_throws_ArgumentNullException_on_null_transactionProcesso
             Action action = () => new AuRaRewardCalculator(_auraParameters, _abiEncoder, null);
             action.Should().Throw<ArgumentNullException>();
         }
+
+        [Test]
+        public void constructor_throws_ArgumentException_on_BlockRewardContractTransition_higher_than_BlockRewardContractTransitions()
```

Comment:
I did not understand the mame of the test.

Reply:
there is an old BlockRewardContractTransition and there is new BlockRewardContractTransitions, they are concatenated, but the singular one (BlockRewardContractTransition) has to be lower than first of the plural one (BlockRewardContractTransitions)

Reply:
the naming is confusing but it is a non-blocker